irods/session.py

```diff
@@ -269,6 +270,9 @@ def configure(self, **kwargs):
     def query(self, *args, **kwargs):
         return Query(self, *args, **kwargs)
 
+    def query2(self, *args, **kwargs):
+        return Query2(self, *args, **kwargs)
```

Comment:
I'm wondering whether it would be better to call this `genquery2`. I understand `query2` is the logical successor to `query` but... I dunno, just feels incomplete to me. :) Any thoughts?

Reply:
Sounds good to me :+1: , have updated the function / class names.